src/modelize/modelize_property.nit

```diff
@@ -951,32 +951,31 @@ redef class AAttrPropdef
 	redef fun build_property(modelbuilder, mclassdef)
 	do
 		var mclass = mclassdef.mclass
+		var nid2 = n_id2
```

Comment:
I praying every day for the removing of `n_id` so this `n_id2` can finally renamed!